### PR TITLE
Update the keywords to make the module more easy to find

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,10 @@
   "keywords": [
     "cli-app",
     "cli",
-    ""
+    "greenkeeper",
+    "bot",
+    "greenkeeper-bot",
+    "friedel"
```
Comment (RichardLitt) [inline package.json]: ahahahahahaha @Dignifiedquire

**Comment by RichardLitt** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (Dignifiedquire) [inline package.json]: what's happening :scream:

Comment (RichardLitt) [inline package.json]: I made a bot to delete notifications from your greenkeeper bot. :D

Comment (diasdavid) [inline package.json]: We are just playing around, ❤️ @Dignifiedquire :)

Comment (Dignifiedquire) [inline package.json]: giphy-3

Don't feed them!

Comment (RichardLitt) [inline package.json]: ... Are you calling me a bear? :P

Comment (RichardLitt) [reply package.json]: This was a good thread.

**Pull request merged**